packages/ui-library/src/components/token-amount-usd-input/index.tsx

```diff
@@ -177,6 +202,30 @@ const UsdInput = ({ onChange, value, token, tokenPrice, onBlur, onFocus, disable
   );
 };
 
+const StyledHeader = styled(ContainerBox).attrs(() => ({ gap: 3, justifyContent: 'center', alignItems: 'center' }))``;
+
+const StyledFooter = styled(ContainerBox).attrs(() => ({ gap: 3, justifyContent: 'center', alignItems: 'center' }))``;
+
+const StyledEndContent = styled(ContainerBox).attrs(() => ({
+  gap: 3,
+  justifyContent: 'space-between',
+  alignItems: 'center',
```

Comment:
maybe we should align `end`? This doesn't match the design (DCA Create):
<img width="221" alt="Screenshot 2024-10-16 at 3 24 21 PM" src="https://github.com/user-attachments/assets/12b88bdf-e679-4b99-85d2-c0a03b621e90">